### ISSUE TITLE
Dragging same item in same ordered list causes duplicate

### ISSUE THREAD

**Issue by matthewpomar** [rapsacnz/DualSelect#2]
Great component. I am trying to implement into my project and I noticed a possible defect. If you drag any item in a given ordered list from top to bottom, it duplicates the item, instead of moving it:

https://www.screencast.com/t/PPjBD4HlDx4

Any ideas where I can fix this?

**Comment by vc4u** [comment]
@matthewpomar  Did you find any workaround for the duplication issue?

**Comment by rapsacnz** [comment]
Can you confirm this is still happening? I thought I'd resolved this one.

**Comment by vc4u** [comment]
Yes it is occurring. The way I resolved it is adding this bit of code in **_handleOnDropParent_** helper method:

```
var tar = event.target;
while(tar !== undefined && tar.tagName !== 'li' && tar.tagName !== 'LI'){
      tar = tar.parentElement;
}
```

And then in the same function replaced:
     `var receivingItem = this.getItem(event.target.id,items);`
with
     `var receivingItem = this.getItem(tar.id,items);`

Looks a dirty fix, but seems to hold good :-) ...

**Comment by rapsacnz** [comment]
A dirty fix is better than no fix! Esp is I'm pushed for time - I'll add it to the code (or you can submit a pull request if you want be be a contributor ;) )

**Comment by vc4u** [comment]
Sure. I'll add my fix :-) ... your component is really good, gr8 work.
